core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/cumSum.kt

```diff
@@ -18,6 +21,36 @@ import kotlin.reflect.typeOf
 
 // region DataColumn
 
+/**
+ * ## The CumSum Operation
+ *
+ * Computes the cumulative sum of the values in the {@get DATA_TYPE}.
+ *
+ * __NOTE:__ If the column contains nullable values and `skipNA` is set to `true`,
+ * skips null values when computing the cumulative sum.
+ * Otherwise, any null value encountered will propagate null values in the output from that point onward.
+ *
+ * {@get [CUMSUM_PARAM] @param [columns]
+ * The names of the columns to apply cumSum operation.}
+ *
+ * @param [skipNA] Whether to skip null values (default: `true`).
+ *
+ * @return A new {@get DATA_TYPE} of the same type with the cumulative sum of the values.
+ *
+ * {@get [CUMSUM_PARAM] @see [Selecting Columns][SelectSelectingOptions].}
+ * @see {@include [DocumentationUrls.CumSum]}
+ */
+@ExcludeFromSources
+@Suppress("ClassName")
+private interface CumSumDocs {
+    interface CUMSUM_PARAM
```

Comment:
It's a bit odd you created an argument interface for `CUMSUM_PARAM` but not for `DATA_TYPE`. Best to probably also add an interface for that for better refactorability :)